build-parent/pom.xml

```diff
@@ -599,7 +599,7 @@
                                     </args>
                                     <buildx>
                                         <builderName>default</builderName>
-                                        <dockerStateDir>~/.docker</dockerStateDir>
+                                        <dockerStateDir>~/.docker/fabric8</dockerStateDir>
                                         <platforms>
```

Comment:
i: providing a separate .docker directory so fabric8 uses the default location but does not override/delete the default `~.docker/config.json`